components/admin_console/system_users/list/selectors.jsx

```diff
@@ -41,3 +41,9 @@ export function getUsers(state, loading, teamId, term, filter) {
 
     return getProfilesInTeam(state, teamId, filters);
 }
+
+export function getNonBotUsers(state, loading, teamId, term, filter) {
```

Comment:
You moved this to a function that is a "selector" but what in meant, is to use a reselect selector, because the whole point here is to cache the `NonBotUsers` based on the changes in the `Users`, but now I'm realizing that the `getUsers` isn't a reselect selector itself, so the whole point of making it a selector makes no sense. I think we can keep this is is now and we can revisit it later.